src/app/content/App/Subscriptions/withConnect.ts

```diff
@@ -1,22 +1,22 @@
 import { connect } from 'react-redux';
-import { Dispatch } from 'redux';
 import { optionsRequested } from 'app/actions/options';
+import clickContributor from '../../actions/goToContributor';
 import { ContentState } from '../../store';
 import {
   getNbTotalContributors,
   getSubscriptions
-} from 'app/options/store/selectors/contributors.selectors';
-import { push } from 'connected-react-router';
+} from 'app/store/selectors/contributors.selectors';
+
+const openSubscriptions = () => optionsRequested();
```

Comment:
This can be simplified

Reply:
Nop, or an event would be passed as an argument to the fn, and this would make the fn optionsRequested breaks.

Reply:
True ! My bad !